arangod/RocksDBEngine/RocksDBCollection.cpp

```diff
@@ -1047,7 +1047,7 @@ Result RocksDBCollection::readFromSnapshot(
   }
 
   return lookupDocumentVPack(
-      trx, token, cb, /*withCache*/ true, readOwnWrites,
+      trx, token, cb, /*withCache*/ false, readOwnWrites,
```

Comment:
What was the motivation for changing the value of the `withCache` parameter here?
Not questioning that it is correct, but would like to understand why it was changed.

Reply:
It feels strange for me that we populate cache with probably outdated results from snapshot.
It used only in inverted index now and in case when MultiGet disabled